converters/fromZigbee.js

```diff
@@ -4191,6 +4191,39 @@ const converters = {
             }
         },
     },
+    tuya_air_housekeeper: {
```

Comment:
Since most of this is duplication of `fz.tuya_air_quality` could we merge both and do a check on the `manufacturerName` to use the correct data points? e.g. `if (meta.device.manufacturerName === '_TZE200_dwcarsat')`

Reply:
Yes, I can modify this.